src/tox/config.py

```diff
@@ -1132,6 +1144,7 @@ def make_envconfig(self, name, section, subs, config, replace=True):
 
     def _getenvdata(self, reader, config):
         candidates = (
+            os.environ.get("_PARALLEL_TOXENV"),
```

Comment:
Like already mentioned: this can be public but should have `TOX_` prefix then.

Reply:
the user already has TOXENV if they want public, this is solely an internal implementation detail and with a special meaning 🤔

Reply:
I meant that all env vars to do with tox should use the prefix `TOX_` so in this case then `TOX_PARALLEL` or something like that.

Reply:
I don't think this should only be considered an implementation detail, as external resources (databases, Redis access...) must be configured accordingly to avoid collisions when toxenvs run in parallel, and we are not just dealing with proper isolated unit tests.

Reply:
I'm still on the opinion that the only safe implementation for databases/Redis access is always parallel, rather than a parallel and a non-parallel way. I would expect these two to have the same effect and work equally well: 

- ''tox -e py27,py37 --parallel``,
- ''bash -c 'tox -e py27' &; bash -c 'tox -e py37' &``.